link_shortener/commands/update.py

```diff
@@ -43,23 +43,25 @@ async def update_link(request, link_id, data):
 
         link_update = links.update().where(links.columns['id'] == link_id)
 
-        if data['password']:
-            salt = os.urandom(32)
-            password = hashlib.pbkdf2_hmac(
-                'sha256',
-                data['password'].encode('utf-8'),
-                salt,
-                100000
-            )
+        if data['password'] != 20 * '\u25CF':
```

Comment:
Dá se tenhle if otočit? Nejdřív něco udělat když se to rovná a zbytek dát do else, bylo by to čitelnější

Reply:
f1c18108902539bf766c546f82298c60ac8c5b16